src/Workspaces/CoreTest/FindReferencesTests.cs

```diff
@@ -268,6 +268,68 @@ class B : C, A
             Assert.Empty(expectedMatchedLines);
         }
 
+        [Fact, WorkItem(18963, "https://github.com/dotnet/roslyn/issues/18963")]
+        public async Task FindReferences_Deconstruction()
```

Comment:
Wrong location for these tests.  Look for the more declarative FindReferencesTests written in VB.